src/patterns/dosdp-patterns/epithelialCellPartOfAnatomicalEntity.yaml

```diff
@@ -0,0 +1,44 @@
+pattern_name: epithelialCellPartOfAnatomicalEntity
+pattern_iri: http://purl.obolibrary.org/obo/cl/epithelialCellPartOfAnatomicalEntity
+
+description: An epithelial cell that is part of an anatomical entity. 
```

Comment:
This is subsumed by the generic C part-of A pattern above

What is our strategy re specificity @matentzn ?

We need to exercise judgment in bringing in patternizer results. They are suggestions